3rd_party/ros/Micro-ROS/Raspberry_PI/Raspberry_ROS_setup.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
```

Comment:
Is there anything Raspberry-pi specific in this script? if not would be best to give it a generic name so it can use against a laptop as well.

Reply:
Deleted.